src/commands/diagnostics.ts

```diff
@@ -0,0 +1,5 @@
+export = {
+	command: 'diagnostics',
+	desc: 'Manage SkySync Diagnostics',
```

Comment:
the description should probably be something like View Skysync Diagnostics.  i don't see anything pertaining to editing / creating / deleting in this story.  we can always update this later when we add additional features.

Reply:
There are a few other diagnostic endpoints for deleting, updating log config, etc, but I'm fine with just changing to 'View' for now until these other commands are available in the cli. We will just have to remember to change it back.